src/src/AdminBundle/Form/EventForm.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace AdminBundle\Form;
+
+use AdminBundle\Entity\Event;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+class EventForm extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('address', TextType::class)
+            ->add('date_time', DateTimeType::class)
+            ->add('notification_threshold', DateTimeType::class)
```

Comment:
AFAK tu ma byt `IntegerType::class`, nie?

Reply:
nie. pozri si model databazy

Reply:
Vsak prave. https://github.com/TIS2017/Nobelisti/pull/68/files#diff-160a30a0116bb7aa66dd3d4db051ece0R61

Reply:
Mas naozaj pravdu. Som to opravila.